### ISSUE TITLE
[COGNITION]: HCA - Heading levels could be adjusted for proper nesting

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#3219]
## Description

The headings for HCA alerts like an applicant not qualifying for VA healthcare or missing financial listings could be improved by changing the heading level fours to heading level threes. Screenshot attached below.

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/1082

## Acceptance Criteria

As a screen reader user, I want to hear heading levels announced in proper nesting order for the main content area.

## Environment

* https://staging.va.gov/health-care/apply/application/introduction
* Test user 371 (or any other test user in the HCA test plan)

## WCAG or Vendor Guidance (optional)

* [Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.2/heading-order)

---
![Screen Shot 2019-11-07 at 11 19 04 AM](https://user-images.githubusercontent.com/934879/68422514-f6133500-0165-11ea-80e7-369a6ebcdcfd.png)
